sbol_utilities/sbol3_sbol2_conversion.py

```diff
@@ -73,6 +73,10 @@ def _convert_identified(self, obj3: sbol3.Identified, obj2: sbol2.Identified):
         obj2.description = self._value_or_property(obj3, obj3.description, 'http://purl.org/dc/terms/description')
         obj2.wasDerivedFrom = obj3.derived_from
         obj2.wasGeneratedBy = obj3.generated_by
+
+        # Add version to identity
+        obj2.identity = obj2.identity + "/" + obj3.sbol2_version
```

Comment:
I don't think it's safe to assume that the `sbol2_version` property is necessarily set.